server/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "back-end-project-week",
+  "version": "1.0.0",
+  "main": "server.js",
+  "scripts": {
+    "start": "nodemon server.js"
+  },
+  "repository": "https://github.com/rushman7/back-end-project-week.git",
+  "author": "<arthurpisakhov1@gmail.com>",
+  "license": "MIT",
+  "dependencies": {
```

Comment:
comprehensive list of dependencies, shows that you're planning ahead.  Nice work!